src/views/HashtagPostView/index.tsx

```diff
@@ -81,11 +85,31 @@ const HashtagPostView = ({ pageData, seo }: HashtagProps): JSX.Element => {
       <Container py={4}>
         <Box textAlign="center">
           <Heading>{pageData?.hashtag?.title}</Heading>
-          <Text my={4} maxW="container.md" mx="auto">
-            {pageData?.hashtag?.content}
-          </Text>
+          <Collapse startingHeight={50} in={show}>
+            <Text my={4} maxW="container.md" mx="auto">
+              {pageData?.hashtag?.content}{' '}
+            </Text>
+          </Collapse>
+          {show ? (
+            <IconButton
+              variant="ghost"
+              size="sm"
+              icon={<ChevronUpIcon />}
+              aria-label="up"
+              bg={'transparent'}
+              _hover={{ color: 'green.400' }}
+              onClick={handleToggle}
+            />
+          ) : (
+            <IconButton
+              icon={<ChevronDownIcon />}
+              aria-label="down"
+              bg={'transparent'}
+              _hover={{ color: 'green.400' }}
+              onClick={handleToggle}
+            />
+          )}
```

Comment:
For those cases I would prefer this method to avoid code duplication:

```suggestion
            <IconButton
              variant="ghost"
              size="sm"
              icon={show ? <FaChevronUp /> : <FaChevronDown />}
              aria-label={show ? "up" : "down"}
              bg={'transparent'}
              _hover={{ color: 'green.400' }}
              onClick={handleToggle}
            />
```